common/focuses/wc_magic_lifestyle_focuses.txt

```diff
@@ -0,0 +1,1453 @@
+﻿light_magic_focus_1 = {
+	lifestyle = light_magic_lifestyle
+
+	is_valid = {
+		can_openly_learn_light_magic_trigger = yes
+	}
+
+	modifier = { }
+
+	desc = {
+		desc = light_magic_focus_1_desc
+		desc = line_break
+	}
+
+	auto_selection_weight = {
+		value = 11
+
+		# Magic trait
+		if = {
+			limit = { has_trait = magic_good_noninheritable }
+			add = 1989
```

Comment:
Why congenital traits not buff it?
<details>
<summary>Click to expand</summary>

![Uploading image.png…]()

</details>

Reply:
> Why congenital traits not buff it?

Congenital traits represent Sorcerers

<https://wow.gamepedia.com/Sorcerer>

> Sorcerers own great talent for arcane spellcasting, but ***they have not harnessed it through the intensive training*** that wizards pursue.

Non-congenial represent Wizards

> Wizards are arcane spellcasters who have undertaken intensive study of magic in places such as the Violet Citadel of Dalaran or the Academies of Silvermoon in Quel'Thalas. They do not channel arcane power, but instead ***use their vast knowledge of otherworldly forces to harness the energies to their will.***

<https://wow.gamepedia.com/Wizard>

On the other branch it's made so sorcerers have more mana and bonuses from unspent mana by default, while wizards learn magic faster.

Reply:
> Congenital traits represent Sorcerers

Jaina is a sorcerer. He had a natural talent and that's why she was sent to Dalaran.

Reply:
> Jaina is a sorcerer.

As well as a wizard. Added trait to her and few other characters in d8465a43

> She had a natural talent and that's why she was sent to Dalaran.

Right now there is no mechanics to send your kids to study magic under another character, so it doesn't really matter.

> Right now, she wouldn't prefer to study Arcane magic at all.

And if I add congenital traits, she will not prefer to study Arcane magic at all anyway, because tirani religion ignore arcane magic.

Reply:
> And if I add congenital traits, she will not prefer to study Arcane magic at all anyway, because tirani religion ignore arcane magic.

True, need to change it. Anyway, she wouldn't even study hydromancy like her father.

Reply:
True, then I need to change it.

Reply:
Allow Arcane magic then too.

Reply:
Done